tests/console/hwloc_testsuite.pm

```diff
@@ -0,0 +1,36 @@
+# SUSE's hwloc tests
+#
+# Copyright © 2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Run hwloc testsuite
+# Maintainer: Thomas Blume <tblume@suse.com>
+
+use base "consoletest";
+use warnings;
+use strict;
+use testapi;
+
+sub run() {
+    # install hwloc testsuite
+    script_run('zypper ar http://download.suse.de/ibs/QA:/SLE12SP2/update/ hwloc-testrepo');
+    script_run('zypper ar http://download.suse.de/ibs/Devel:/HPC:/SLE12SP2/standard/ HPC-module-hwloc');
+    script_run('zypper --gpg-auto-import-keys ref');
+    assert_script_run('zypper -n in hwloc hwloc-testsuite');
+
+    # run the testsuite test scripts
+    assert_script_run('cd /var/opt/hwloc-tests && ./run-tests.sh');
```

Comment:
@asmorodskyi added a test module recently which should test something which is very similar of what you are trying to achieve here. See https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/3136/files for details. It would be great if you could incorporate a similar workflow. Feel free to talk to @asmorodskyi as well, same IRC nick as well